pkg/channel/distributed/dispatcher/dispatcher/handler.go

```diff
@@ -175,7 +176,7 @@ func (h *Handler) checkRetry(_ context.Context, response *http.Response, err err
 	//        status codes from the subscriber and returning 400s instead.  Once this has,
 	//        been resolved we can remove 400 from the list of codes to retry.
 	//
-	if statusCode >= 500 || statusCode == 400 || statusCode == 404 || statusCode == 429 {
+	if statusCode >= 500 || statusCode == 400 || statusCode == 404 || statusCode == 429 || statusCode == 409 {
```

Comment:
was there some `Conflict` ?

Reply:
Historically some of these codes were to workaround broker/filter "bugs" and I believe the new "409" is because that is what the eventing retry e2e test is sending and expecting a retry to happen - Eric can confirm.

Reply:
The TestBrokerRedelivery fails for the distributed channel because eventing/test/test_images/recordevents returns http.StatusConflict (409) when given a skip algorithm.  The distributed e2e tests are not quite finished yet for some other reasons but this file was already being changed so I left this pre-emptive fix in place.  It can be moved to the e2e PR if that would be cleaner.

Reply:
@eric-sap I would prefer if you move this particular change to the e2e pr